### DISCUSSION TITLE
Overlapping renders with transparency?

### DISCUSSION THREAD

**Discussion by TomJGooding** [Textualize/textual#5300] [Q&A] [answered]
This might not be possible or I'm missing something obvious, but is there a way of overlapping widget renders with transparency? Here's a contrived example which hopefully demonstrates what I mean:

```python
from textual.app import App, ComposeResult, RenderResult
from textual.widget import Widget


class O_e_l_p_i_g(Widget):
    def render(self) -> RenderResult:
        return "O e l p i g"


class _v_r_a_p_n_(Widget):
    def render(self) -> RenderResult:
        return " v r a p n "


class OverlappingApp(App):
    CSS = """
    Widget {
        height: auto;
        position: absolute;
    }
    """

    def compose(self) -> ComposeResult:
        yield _v_r_a_p_n_()
        yield O_e_l_p_i_g()


if __name__ == "__main__":
    app = OverlappingApp()
    app.run()
```

**Comment by willmcgugan** [comment, marked as answer]
Not like that, no. The compositor treats each render as a solid strip.

**Reply by TomJGooding** [reply]
Thanks Will for the quick reply! I thought that was probably the case but wanted to check. I doubt that would have a use case anyway outside of my toy project to experiment with Textual.

**Reply by TomJGooding** [reply]
@willmcgugan Here's my toy project if you're curious and need a break from fixing issues! https://github.com/TomJGooding/textual-dino

**Reply by willmcgugan** [reply]
Ha! Was not expecting that. Bravo!
